core/src/test/scala/unit/kafka/server/AbstractFetcherThreadTest.scala

```diff
@@ -25,6 +25,7 @@ import com.yammer.metrics.Metrics
 import kafka.cluster.BrokerEndPoint
 import kafka.log.LogAppendInfo
 import kafka.message.NoCompressionCodec
+import kafka.server.AbstractFetcherThread.ReplicaFetch
```

Comment:
unused import

Reply:
I believe it's used here https://github.com/apache/kafka/pull/7443/files/4a306ad7967e20712ec2c0ea085a91b5f471b9a3#diff-f84579dc9ecc952780289986cddecc46R905. It's not showing up as a checkstyle error for me.